Jenkinsfile_tactical

```diff
@@ -0,0 +1,33 @@
+#!groovy
+
+@Library("Reform") _
+
+timestamps {
+    milestone()
+    lock(resource: "service-auth-provider-app-${env.BRANCH_NAME}", inversePrecedence: true) {
+        node {
+            try {
+                stage('Checkout') {
+                    deleteDir()
+                    checkout scm
+                }
+
+                stage('Build') {
+                  sh "./gradlew installDist"
+                }
+
+                onMaster {
+                    stage('Publish Docker') {
+                        dockerImage imageName: 'auth/service-auth-provider-api'
```

Comment:
I have few of queries on this one.
Do we need `Jenkins_tactical` to publish images? 
When it this file going to be picked up by pipeline?
Docker Image name is same as one used by tactical service auth provider.Do we need to use a different image name?
At what stage are we publishing the image?

Reply:
We do for now, I discussed this with Jason on Friday and the strategic option we will hopefully be exploring next sprint will be publishing to docker hub.
But I would like to migrate to the new s2s version before that, if you don't want this added I can publish the latest version now manually so that we can migrate to that.

This file will be picked up on the tactical jenkins only.

No we don't need to rename its the same app, I'll make sure the old one is disabled though so they don't clash

After building the jar

Reply:
https://git.reform.hmcts.net/reform-idam/service-auth-provider-app/pull/59

Reply:
Thanks all good from my side :)